tests/test_admin_fields.py

```diff
@@ -0,0 +1,39 @@
+from cms.api import add_plugin
+
+from djangocms_versioning.constants import DRAFT, PUBLISHED
+from djangocms_versioning.models import Version
+
+from djangocms_versioning_filer.models import FileGrouper
+
+from .base import BaseFilerVersioningTestCase
+
+
+class AdmitFieldsMonkeypatchTest(BaseFilerVersioningTestCase):
```

Comment:
This test may now be redundant, if not: Typo on Admin and it's a TestCase not a unit test "AdmitFieldsMonkeypatchTest" should be "AdminFieldsMonkeypatchTestCase"

Reply:
This file is now deleted as it was testing the functionality of publishing drafts automatically.